airbyte-cdk/python/airbyte_cdk/sources/declarative/manifest_declarative_source.py

```diff
@@ -127,13 +132,33 @@ def _validate_source(self):
         if len(streams) > 0:
             full_config["streams"] = streams
         declarative_source_schema = ConcreteDeclarativeSource.json_schema()
-        validate(full_config, declarative_source_schema)
+
+        try:
+            validate(full_config, declarative_source_schema)
+        except ValidationError as e:
+            raise ValidationError("Validation against auto-generated schema failed") from e
+
+        # Validates the connector manifest against the low-code component json schema
+        manifest = self._source_config
+        manifest_transformer = ManifestComponentTransformer()
+        propagated_manifest = manifest_transformer.propagate_types_and_options("", manifest, {})
+
+        try:
+            raw_component_schema = pkgutil.get_data(__name__, "low_code_component_schema.yaml")
```

Comment:
I'm a little nervous with these pkgutil loads now. did we confirm everything is properly loaded when running from the platform?

Reply:
fair point, i can try redeploying greenhouse locally to verify. But I'm not sure what alternatives we have for loading files safely beyond `pkgutil`. Using `open()` doesn't work from a container last time we tried.

Reply:
I'm 100% fine using pkgutil - just want to make sure we run a test with the platform before we merge

Reply:
I agree and I just ran this on a local instance of Airbyte for greenhouse and the spec operation runs correctly so we're able to read and parse the yaml file correctly using pkgutil e2e